File: libvirt/c/basic.c
#include <stdio.h>
#include <libvirt/libvirt.h>

int main(int argc, char *argv[])
{
  virConnectPtr conn;
  conn = virConnectOpen("qemu:///system");
  if (conn == NULL) {
    fprintf(stderr, "Failed to open connection to qemu:///system\n");
    return 1;
  }

  printf(virConnectGetCapabilities(conn));
  
  virConnectClose(conn);
  return 0;
}

File: libvirt/go/basic.go
package main

import (
	"fmt";
	libvirt "libvirt.org/libvirt-go"
)

func main() {
	conn, err := libvirt.NewConnect("qemu:///system")
	if err != nil {
		fmt.Printf("Failed to open connection to QEMU")
	}
	
	fmt.Printf(conn.GetCapabilities())
	
	defer conn.Close()
}
